gui/src/hooks/config.ts

```diff
@@ -86,18 +100,16 @@ export function useConfigProvider(): ConfigContext {
     loadConfig: async () => {
       setLoading(true);
       try {
-        const migrated = localStorage.getItem('configMigrated');
+        const migrated = await localStorage.getItem('configMigratedToTauri');
         if (!migrated) {
-          const oldConfig = await readTextFile('config.json', {
-            dir: BaseDirectory.AppConfig,
-          }).catch(() => null);
+          const oldConfig = localStorage.getItem('config.json');
 
-          if (oldConfig) localStorage.setItem('config.json', oldConfig);
+          if (oldConfig) await store.set('config.json', oldConfig);
 
-          localStorage.setItem('configMigrated', 'true');
+          localStorage.setItem('configMigratedToTauri', 'true');
         }
```

Comment:
Shouldn't you be setting `configMigratedToTauri` in the `store` config, not `localStorage`?

Reply:
i guess it really doesnt matter haha

Reply:
i will use the `store` then